marklogic-data-hub-central/ui/src/components/queries/queries.tsx

```diff
@@ -65,7 +65,8 @@ const Query = (props) => {
                     entityTypeIds: searchOptions.entityTypeIds.length ? searchOptions.entityTypeIds : props.entities,
                     selectedFacets: facets,
                 },
-                propertiesToDisplay: props.columns,
+                // propertiesToDisplay: props.columns,
```

Comment:
Can we delete these commented out lines.

Reply:
Removed.